src/bigdecimal.ts

```diff
@@ -2956,6 +2967,101 @@ export class BigDecimal {
         }
     }
 
+    /**
+    * Alias for `compareTo(val) === 0`.
+    * Consider using {@link equals} in case the precision needs to be considered.
+    * @returns true if the value is the same as `val`
+    * @see {@link equals}
+    * @see     {@link compareTo}
+    */
+    sameValue(val: BigDecimal | BigInt | number | string): boolean {
```

Comment:
```suggestion
    sameValue(val: BigDecimal | bigint | number | string): boolean {
```

this is needed because I upgraded eslint on master recently.

Reply:
Fixed in ea7d2af0f5e023248d1599c320ba16d5fe312ec8